src/AutomatedCar/SystemComponents/Packets/HMIPacket.cs

```diff
@@ -150,5 +150,45 @@ public void AccSet(bool on)
                 this.acc = !this.acc;
             }
         }
+
+        public void HandleGasPedal(bool isGasPedalDown)
+        {
+            if (isGasPedalDown)
+            {
+                Increase(ref this.gaspedal);
+            }
+            else
+            {
+                Decrease(ref this.gaspedal);
+            }
+        }
+
+        public void HandleBrakePedal(bool isBrakePedalDown)
+        {
+            if (isBrakePedalDown)
+            {
+                Increase(ref this.gaspedal);
+            }
+            else
+            {
+                Decrease(ref this.gaspedal);
+            }
+        }
+
+        private void Increase(ref double pedal)
+        {
+            if (pedal < 100)
+            {
+                pedal += 1 / 60;
```

Comment:
Az értéket rosszul mondtam, 10/6-al kell növelni illetve csökkenteni, ha azt akarjuk hogy 1 mp alatt legyen 0-ról 100-on a pedál értéke.
De  most jut eszembe, hogy az lett eldöntve hogy egyenlőre 1 mp a gázpedál és 0.5 mp a fékpedál 0-100 ideje. Ha mindenképp egy Increase és egy Decrease metódust akarsz használni akkor bemenetnek lehetne egy double változó ami  10/(mp)*6 logika alapján lesz megadva, a féknél ha meghívod akkor pl 
`Increase(ref this.gaspedal, 10/3);`
gáznál pedig 
`Increase(ref this.gaspedal, 10/6);`

Reply:
okes, ezeket gyorsan bele is rakom